client/utils.go

```diff
@@ -90,6 +91,46 @@ func authorsName(a database.Author) string {
 	return displayName
 }
 
+type DarkmodeStatus struct {
+	Darkmode bool
+	LastPoll time.Time
+	Url      string
+}
+
+// darkmode checks if the mörkläggning is active by making request to
+// an external API. It parses and outputs the result as a bool.
+// If any error occurs during the request or parsing the response, it returns
+// the default dark mode status which is true.
+func Darkmode(ds *DarkmodeStatus) bool {
```

Comment:
This might be fine, but this can get called at the same time from multiple threads, so there _should_ perhaps be a mutex locking this function.

Reply:
Good call. A mutex has been added.